make/test/BuildMicrobenchmark.gmk

```diff
@@ -96,6 +96,15 @@ $(eval $(call SetupJavaCompilation, BUILD_JDK_MICROBENCHMARK, \
     BIN := $(MICROBENCHMARK_CLASSES), \
     JAVAC_FLAGS := --add-exports java.base/sun.security.util=ALL-UNNAMED \
         --add-exports java.base/sun.invoke.util=ALL-UNNAMED \
+        --add-exports java.base/jdk.classfile=ALL-UNNAMED \
+        --add-exports java.base/jdk.classfile.attribute=ALL-UNNAMED \
+        --add-exports java.base/jdk.classfile.constantpool=ALL-UNNAMED \
+        --add-exports java.base/jdk.classfile.instruction=ALL-UNNAMED \
+        --add-exports java.base/jdk.classfile.jdktypes=ALL-UNNAMED \
+        --add-exports java.base/jdk.classfile.components=ALL-UNNAMED \
+        --add-exports java.base/jdk.classfile.impl=ALL-UNNAMED \
+        --add-exports java.base/jdk.internal.org.objectweb.asm=ALL-UNNAMED \
```

Comment:
Why do you need to export ASM now, when it was not needed before? I thought the purpose here was to replace ASM..?

Reply:
There are new benchmarks added, comparing performance of Classfile API with ASM.

Reply:
I see.